test/behavior/bugs/12680.zig

```diff
@@ -0,0 +1,10 @@
+// export this function twice
+pub export fn testFunc() callconv(.C) void {}
+
+comptime {
+    @export(testFunc, .{ .name = "test_func", .linkage = .Strong });
+}
+
+test "export a function twice" {
+    _ = testFunc();
```

Comment:
This doesn't really test that it is being exported correctly. Try putting the export in a separate function and an extern in this one and check that it is returning the correct value.